Updated changelog

amazzini opened the pull request.

Commit:
Updated changelog

Commit:
Some final adjustments for 2.4.1

- TapiFm: DetectedCondition augments the new ActiveCondition, composed
by the new FaultManagementContext.
- TapiOam: identity wrongly in package diagrams
- TapiTopology/Notification/PathComp/Oam/VirtualNtw: Comments

Commit:
Updated changelog

Commit:
Further minor refinements

Commit:
Removal of Gendoc directory

Commit:
Remove of CHANGE_LOG directory

Commit:
Updated the YANG revision dates

nigel-r-davis reviewed (approved):
I have reviewed all changes in Yang, UML and structure. I saw nothing unexpected and nothing appeared to be in error.

Commit:
Last commit for 2.4.1

TapiPhotonic: readonly on PowerProperties/powerSpectralDensity.
Published TR-547-TAPI Reference Implementation Agreement_v2.1
Published TapiGenDoc_v2.4.1
Published TAPI_Alarm_TCA_List_v2.1.0
Published HighLevelDiff_Tapi2.1.3To2.4.1

Commit:
last changelog for 2.4.1

nigel-r-davis commented:
The latest change is also approved.
